s3tests/functional/test_s3.py

```diff
@@ -4926,7 +4926,23 @@ def _check_key_content(src, dst):
 
 @attr(resource='object')
 @attr(method='put')
-@attr(operation='check multipart copies with single small part')
+@attr(operation='check multipart copies with single small part - ranges not NotImplemented')
+@attr(assertion='fails')
+@attr('fails_on_aws')
+def test_multipart_copy_small_notimplemented():
+    (src_bucket, src_key) = _create_key_with_random_content('foo')
+    dst_bucket = get_new_bucket()
+    dst_keyname = "mymultipart"
+    size = 1
+    e = assert_raises(boto.exception.S3ResponseError, _multipart_copy, src_bucket.name, src_key.name, dst_bucket, dst_keyname, size)
+    eq(e.status, 501)
+    eq(e.error_code, u'NotImplemented')
```

Comment:
Why is it so important to test the temporary rgw `NotImplemented` code path?  Other S3 implementations like S3Proxy and akashic use s3-tests and this clutter is just another thing for us to work around.  Why not just annotate with `fails_on_rgw` and be done with it?